.buildkite/pipeline-resource-definitions/kibana-purge-cloud-deployment.yml

```diff
@@ -5,7 +5,7 @@ metadata:
   name: bk-purge-cloud-deployment
   description: Purge a cloud deployment
   links:
-    - url: 'https://buildkite.com/elastic/purge-cloud-deployment'
+    - url: 'https://buildkite.com/elastic/kibana-purge-cloud-deployment'
```

Comment:
We can delete this one, or I can follow up later.  It was superseded by the `ci:cloud-redeploy` label.

Reply:
Also, the referenced pipeline impl (https://github.com/elastic/kibana/blob/dc664bbf0b9cf3b2ef13d2978ed616dcbf5b4b85/.buildkite/pipelines/purge_cloud_deployment.yml), or just this pipeline?

Reply:
Oh, yes both is fine.  We don't need any of the supporting files anymore.

Reply:
cleaned all related files: 2fc7ea826a3